### PR TITLE
fix(service-portal): Print screens + simple filter

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

Little addition to simple filter.
Add print button to all finance screens.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx [hunk 1]
```diff
@@ -114,49 +115,74 @@ const DocumentScreen: FC<Props> = ({
...
+                  variant="utility"
+                >
+                  {formatMessage(m.print)}
+                </Button>
+              </GridColumn>
+            </GridRow>
+            <Box marginTop={3}>
+              <Input
+                label="Leit"
+                name="Search1"
```
Comment (osk) [inline libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx:177]: Why not `Search`?

File: libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx [hunk 2]
```diff
@@ -114,49 +115,74 @@ const DocumentScreen: FC<Props> = ({
...
+                placeholder="Sláðu inn leitarorð"
```
Comment (osk) [inline libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx:178]: Translation

File: libs/service-portal/finance/src/screens/FinanceTransactions/FinanceTransactions.tsx [hunk 3]
```diff
@@ -145,70 +160,70 @@ const FinanceTransactions: ServicePortalModuleComponent = ({ userInfo }) => {
...
-              <Select
-                name="faerslur"
-                backgroundColor="blue"
-                placeholder={formatMessage(m.transactions)}
-                label={formatMessage(m.transactionsLabel)}
-                defaultValue={allChargeTypes}
+            <Box marginTop={3}>
+              <Input
+                label="Leit"
+                name="Search1"
```
Comment (osk) [inline libs/service-portal/finance/src/screens/FinanceTransactions/FinanceTransactions.tsx:218]: `Search`?

File: libs/service-portal/finance/src/screens/FinanceTransactions/FinanceTransactions.tsx [hunk 4]
```diff
@@ -145,70 +160,70 @@ const FinanceTransactions: ServicePortalModuleComponent = ({ userInfo }) => {
...
+                placeholder="Sláðu inn leitarorð"
```
Comment (osk) [inline libs/service-portal/finance/src/screens/FinanceTransactions/FinanceTransactions.tsx:219]: Translation

**Review by osk — CHANGES_REQUESTED** [review]

File: libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx (hunk 1 above)
Comment (thordurhhh) [reply libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx]: No idea why I did that, this was added a few weeks ago. 

Fixed!

**Review by osk — APPROVED** [review]

**Pull request merged**